designs/Net5PlatformAndPlatformVersion.md

```diff
@@ -0,0 +1,177 @@
+# net5.0 TFM Platform and PlatformVersion implementation
+
+This document is meant to describe the various parts of the code that need to be updated in order to support platforms in net5.0 target framework identifiers. That is, the optional part after a `-` following `net5.0`. For example, `net5.0-ios14.1`. Note that this ONLY covers the platform part, because [initial support for `net5.0` has already been added](https://github.com/NuGet/NuGet.Client/pull/3265).
+
+For more details on the feature itself, please refer to [the full spec](https://github.com/terrajobst/designs/blob/master/accepted/2020/net5/net5.md).
+
+## class NuGetFramework
+
+Source: src/NuGet.Core/NuGet.Frameworks/NuGetFramework.cs
+
+### Fields
+
+Three additional fields will be added:
+
+```csharp
+        /// <summary>
+        /// Framework Platform (net5.0+)
+        /// </summary>
+        public string Platform

```

Comment:
Suggestion: maybe a different type Platform - to include the name and version?

Reply:
Doing it this way makes it consistent with how TFI and TFV are handled, though, so I prefer the split version. Also makes those fields more straightforward to access.